### DISCUSSION TITLE
New suggestion like Clipy right-click menu

### DISCUSSION THREAD

**Discussion by codeshif** [massCodeIO/massCode#53] [Ideas]
Hello, I am also a developer. 
The software you make is like SnippetsLab, which is unique to the Mac platform. I am also a loyal user of it, but it is a pity that there is no Windows version. Thank you very much for making such excellent cross-platform software.
I have a careful mind. If the current massCode is combined with the idea of Clipy, an open source software for the Mac platform, it will be invincible.
I don't know if you know Clipy, please check it out.
https://github.com/Clipy/Clipy
Thanks again for this god-like software, thank you very much.
![image](https://user-images.githubusercontent.com/35602158/165366922-c25bbbd0-e81e-40ff-9fe7-06fe46871f3f.png)

As shown in the figure, Clipy can quickly view and use the clipboard content or template content of the database by using the system hotkey right-click menu. I have tried to modify the code of Clipy to make it more like SnippetsLab, but the technical ability is limited. Hope this is a good suggestion.

Looking forward to your reply.

**Comment by antonreshetov** [comment]
Thanks for the suggestion. The problem is that this is written in Swift and only for macOS, while massCode is a cross-platform application written in JS. In general, Raycast or Alfred solve the problem of expanding the clipboard for macOS.

**Reply by codeshif** [reply]
Thanks for your reply, OK.
